dockerfiles/packaging/guice/cassandra-rabbitmq/package/etc/james/templates/quota.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
```

Comment:
This file is not needed. Please remove it.

Reply:
There is a `quota.xml` in `packaging/guice/cassandra`. I guess I should remove it also